config/fields/users.php

```diff
@@ -24,29 +24,31 @@
 		/**
 		 * Default selected user(s) when a new page/file/user is created
 		 */
-		'default' => function ($default = null) {
-			if ($default === false) {
+		'default' => function (string|array|bool $default = null) {
```

Comment:
Shouldn't the type hints be: 

```suggestion
		'default' => function (string|array|bool|null $default = null) {
```